spec/support/controller_helpers.rb

```diff
@@ -4,8 +4,6 @@
 # Example:
 #   let(:headers) { { token: user.token } }
 RSpec.shared_examples 'an authenticated resource action' do
-  it { is_expected.to have_http_status(:ok) }
```

Comment:
why?

Reply:
I removed this line because each endpoint will have a different http status response.